ApiReview/API.Polly.Core/NoDocs/Polly.Simmy/OutcomeChaosPipelineBuilderExtensions.cs

```diff
@@ -0,0 +1,20 @@
+// Assembly 'Polly.Core'
+
+using System;
+using System.Diagnostics.CodeAnalysis;
+using Polly.Simmy.Outcomes;
+
+namespace Polly.Simmy;
+
+public static class OutcomeChaosPipelineBuilderExtensions
+{
+    public static ResiliencePipelineBuilder AddChaosFault(this ResiliencePipelineBuilder builder, bool enabled, double injectionRate, Exception fault);
+    public static ResiliencePipelineBuilder AddChaosFault(this ResiliencePipelineBuilder builder, bool enabled, double injectionRate, Func<Exception?> faultGenerator);
```

Comment:
Do we need both of these given someone could use the second one and do `static (_) => new BlahException("whatever"))`?

Reply:
It's just for convenience, having those two options for the consumers doesn't hurt I guess.

Reply:
Polly v7 had issues evolving due to the explosion of overloads we had that made adding a new one difficult without breaking stuff - limiting trivial convenience overloads might help us avoid that in the future.

It's still relatively simple here so not a hill I'll die on, but worth considering.

Reply:
I removed the `enabled` parameter FYI so the convenience extensions are now simpler.